extension/src/ctp/ctp.js

```diff
@@ -66,9 +72,19 @@ function setUpClient () {
         this.buildRequestOptions(uri.byId(id).build(), 'POST', body)
       )
     },
+
     fetch (uri) {
       return ctpClient.execute(this.buildRequestOptions(uri.build()))
     },
+
+    fetchBatches (uri, cb, opts = { accumulate: false }) {
```

Comment:
what is `cb`?

Reply:
callback

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/5/commits/e2695da09020b78e0188bb10bdc741bfa88aa4f4